src/queries.js

```diff
@@ -11,6 +11,47 @@ const influx = new Influx.InfluxDB({
   port: process.env.INFLUX_PORT || '8086'
 })
 
+// merge the MEAN values from month and year queries to accept http and https link as the same key;
+const merge_results = (seriesRows, days) => {
+  const seriesValues = new Object()
+  const seriesDuplicates = {};
+
+  for (const row of seriesRows) {
+    const url = row.url.split('//')[1];
```

Comment:
This looks pretty fragile. Isn't there another way of separating the protocol part from the url?